src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleExtensionId.java

```diff
@@ -60,6 +60,7 @@ public static IsolationKey create(ModuleKey module, String usageExportedName) {
 
     @Override
     public final String toString() {
+      // NOTE: Can't be bothered to switch this based on the flag.
```

Comment:
I can relate, but add a comment reminding us to change this for Bazel 8.

Reply:
Is this function used widely?

Reply:
no, only used if `--experimental_isolated_module_extensions` is set. added a comment.